Resources/views/standard_layout.html.twig

```diff
@@ -32,30 +33,34 @@ file that was distributed with this source code.
 
         {% block stylesheets %}
             {% for stylesheet in sonata_admin.adminPool.getOption('stylesheets', []) %}
-                    <link rel="stylesheet" href="{{ asset(stylesheet) }}">
+                <link rel="stylesheet" href="{{ asset(stylesheet) }}">
             {% endfor %}
         {% endblock %}
 
-        {% block javascripts %}
-            <script>
-                window.SONATA_CONFIG = {
-                    CONFIRM_EXIT: {% if sonata_admin.adminPool.getOption('confirm_exit') %}true{% else %}false{% endif %},
-                    USE_SELECT2: {% if sonata_admin.adminPool.getOption('use_select2') %}true{% else %}false{% endif %},
-                    USE_ICHECK: {% if sonata_admin.adminPool.getOption('use_icheck') %}true{% else %}false{% endif %},
-                    USE_STICKYFORMS: {% if sonata_admin.adminPool.getOption('use_stickyforms') %}true{% else %}false{% endif %}
-                };
-                window.SONATA_TRANSLATIONS = {
-                    CONFIRM_EXIT:  '{{ 'confirm_exit'|trans({}, 'SonataAdminBundle')|escape('js') }}'
-                };
-
-                // http://getbootstrap.com/getting-started/#support-ie10-width
-                if (navigator.userAgent.match(/IEMobile\/10\.0/)) {
-                    var msViewportStyle = document.createElement('style');
-                    msViewportStyle.appendChild(document.createTextNode('@-ms-viewport{width:auto!important}'));
-                    document.querySelector('head').appendChild(msViewportStyle);
+        <script>
+            window.Sonata = {
+                debug: {{ app.debug|json_encode }},
+                config: {
+                    confirmExit: {{ sonata_admin.adminPool.getOption('confirm_exit') ? 'true' : 'false' }},
+                    useSelect2: {{ sonata_admin.adminPool.getOption('use_select2') ? 'true' : 'false' }},
+                    useICheck: {{ sonata_admin.adminPool.getOption('use_icheck') ? 'true' : 'false' }},
+                    useStickyForms: {{ sonata_admin.adminPool.getOption('use_stickyforms') ? 'true' : 'false' }}
```

Comment:
I personally prefer
```twig
{{ sonata_admin.adminPool.getOption('confirm_exit')|json_encode }}
```
But I wasn't 100% shure what the actual value could be...